### PR TITLE
chore(demo-playwright): writing input-month-range test using playwright

### PR DESCRIPTION
**Pull request by VidhiRambhia**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [x] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Addresses https://github.com/taiga-family/taiga-ui/issues/4912

## What is the new behaviour?

This PR adds an e2e test (with playwright) for input-month-range

### CODE REVIEW

File: projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, Locator, test} from '@playwright/test';
+
+import {TuiInputMonthRangePO} from '../../../utils/page-objects/input-month-range.po';
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts:4]: Add new PO to the following file:
https://github.com/taiga-family/taiga-ui/blob/main/projects/demo-playwright/utils/page-objects/index.ts

And move this import to `@demo-playwright/utils`.

File: projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        beforeEach(({page}) => {
+            documentationPage = new TuiDocumentationPagePO(page);
+            example = documentationPage.apiPageExample;
+            inputMonthRange = new TuiInputMonthRangePO(
+                example.locator(`tui-input-month-range`),
+            );
+        });
+
+        test(`Maximum month less than current month`, async ({page}) => {
+            await tuiGoto(page, `components/input-month-range/API?tuiMode=null&max$=1`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts:27]:
```suggestion
            await tuiGoto(page, `components/input-month-range/API?max$=1`);
```

File: projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            await inputMonthRange.textfield.click();
+
+            await documentationPage.prepareApiPageBeforeScreenshot();
+            await expect(page).toHaveScreenshot(`input-month-range-maximum-month.png`);
+        });
+
+        test(`Minimum month more than current month`, async ({page}) => {
+            await tuiGoto(page, `components/input-month-range/API?tuiMode=null&min$=3`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts:35]:
```suggestion
            await tuiGoto(page, `components/input-month-range/API?min$=3`);
```

Comment (VidhiRambhia) [reply projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts:35]: Done

File: projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts (hunk 2 above)
Comment (VidhiRambhia) [reply projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts:27]: Changed this

File: projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts (hunk 1 above)
Comment (VidhiRambhia) [reply projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts:4]: Changed this

**Review by nsbarsukov — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Pull request merged**